### ISSUE TITLE
bug(sbb-tab-group): focus outline of tab is animated

### ISSUE THREAD

**Issue by jeripeierSBB** [sbb-design-systems/lyne-components#4326]
The tab receives focus which is displayed with the usual focus outline. However, it's position and color seems to be animated. Animation for focus outline should be avoided.
